.gitattributes

```diff
@@ -1 +1,4 @@
 *.sol linguist-language=Solidity
+prerelease.lock export-ignore
```

Comment:
Isn't this file now inside `cmake/` subdir?
```suggestion
/cmake/prerelease.lock export-ignore
```

Reply:
Also, the other files might need the `/` at the beginning if paths here work like in `.gitignore`.